package.json

```diff
@@ -78,7 +78,7 @@
       "sqlite3"
     ],
     "overrides": {
-      "@langchain/core": "0.1.41",
+      "@langchain/core": "0.1.61",
```

Comment:
Can we please extract all langchain upgrades into a separate PR?

Reply:
@netroy Sure. Can you review it then, please? https://github.com/n8n-io/n8n/pull/9252